libs/positions/src/lib/positions-table.tsx

```diff
@@ -113,234 +124,299 @@ export const AmountCell = ({ valueFormatted }: AmountCellProps) => {
+          field="capitalUtilisation"
+          type="rightAligned"
+          flex={2}
+          cellRenderer="ProgressBarCell"
+          cellRendererSelector={(
+            params: ICellRendererParams
+          ): CellRendererSelectorResult => {
+            return {
+              component: params.node.rowPinned ? EmptyCell : ProgressBarCell,
+            };
+          }}
+          valueFormatter={({
+            data,
             value,
-          }: {
-            value: string;
-          }) => value && BigInt(value) < 0,
-        }}
-        valueFormatter={({
-          value,
-          data,
-        }: PositionsTableValueFormatterParams & {
-          value: Position['realisedPNL'];
-        }) =>
-          value === undefined
-            ? undefined
-            : addDecimalsFormatNumber(value.toString(), data.assetDecimals)
-        }
-        cellRenderer="PriceFlashCell"
-        headerTooltip={t('P&L excludes any fees paid.')}
-      />
-      <AgGridColumn
-        headerName={t('Unrealised PNL')}
-        field="unrealisedPNL"
-        type="rightAligned"
-        cellClassRules={{
-          'text-vega-green-dark dark:text-vega-green': ({
+            node,
+          }: PositionsTableValueFormatterParams & {
+            value: Position['capitalUtilisation'];
+          }): PriceCellProps['valueFormatted'] | undefined => {
+            if (!data || node?.rowPinned) {
+              return undefined;
+            }
+            return {
+              low: `${formatNumber(value, 2)}%`,
+              high: addDecimalsFormatNumber(data.totalBalance, data.decimals),
+              value: Number(value),
+            };
+          }}
+        />
+        <AgGridColumn
+          headerName={t('Realised PNL')}
+          field="realisedPNL"
+          type="rightAligned"
+          cellClassRules={{
+            'text-vega-green-dark dark:text-vega-green': ({
+              value,
+            }: {
+              value: string;
+            }) => value && BigInt(value) > 0,
+            'text-vega-red-dark dark:text-vega-red': ({
+              value,
+            }: {
+              value: string;
+            }) => value && BigInt(value) < 0,
+          }}
+          valueFormatter={({
             value,
-          }: {
-            value: string;
-          }) => value && BigInt(value) > 0,
-          'text-vega-red-dark dark:text-vega-red': ({
+            data,
+          }: PositionsTableValueFormatterParams & {
+            value: Position['realisedPNL'];
+          }) =>
+            value === undefined
+              ? undefined
+              : addDecimalsFormatNumber(value.toString(), data.decimals)
+          }
+          cellRenderer="PriceFlashCell"
+          headerTooltip={t('P&L excludes any fees paid.')}
+        />
+        <AgGridColumn
+          headerName={t('Unrealised PNL')}
+          field="unrealisedPNL"
+          type="rightAligned"
+          cellClassRules={{
```

Comment:
chore (non blocking): Would it be worthwhile for this be moved to ui-toolkit/ag-grid?